beacons/src/androidLibAndroidTest/kotlin/TestActivity.kt

```diff
@@ -0,0 +1,23 @@
+/*
+
+Copyright 2019 Splendo Consulting B.V. The Netherlands
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+*/
+
+package com.splendo.kaluga.beacons
```

Comment:
Like ongoing permission work (`.permissions.location` etc), this could be a sub package, `com.splendo.kaluga.bluetooth.beacons`

Reply:
Does it mean I have to move in into `bluetooth/beacons` subfolder?

Reply:
I think this will create gradle issues, I created https://github.com/splendo/kaluga/issues/337 to combine this effort.